metron-interface/metron-alerts/src/app/utils/httpUtil.ts

```diff
@@ -33,9 +33,12 @@ export class HttpUtil {
     return body || {};
   }
 
+  /**
+   * @depricated Turning all errors to 404 and hiding actual errors from the consumers
+   *             could limit how we can recover or react to errors.
+   *             Use sessionExpiration instead and/or introduce new composable error handlers.
+   */
   public static handleError(res: HttpErrorResponse): Observable<RestError> {
-    // In a real world app, we might use a remote logging infrastructure
-    // We'd also dig deeper into the error to get a better message
```

Comment:
If we once accepted to have comments like this in the code using // FIXME is also acceptable. :)